### DISCUSSION TITLE
Charting the game with no under/overchart

### DISCUSSION THREAD

**Discussion by isakube** [FunkinCrew/Funkin#3681] [Show and tell]
Hello Funkin' GitHub,

I'm working on recharting the game. Just for fun, I guess. Not sure if this should be Ideas or Show and Tell, but I'm putting it in Show and Tell because I plan on linking footage of charts in thread.

In vanilla, Easy and Normal difficulties are made easier than Hard difficulty via undercharting (deliberately removing notes from the rhythm to make the chart easier to play.) This is... not ideal, as in my opinion it makes the charts feel a little too empty, even at EZ/NM's lower scroll speeds, not to mention the resulting horrendous song representation. My recharting project intends to fix this by keeping every note the same rhythmically, but changing the PATTERNS. Specifically, changing the patterns to be more friendly to WASD or other one-handed control schemes, which ninjamuffin has declared is ["the way god intended".](https://twitter.com/ninja_muffin99/status/1837525854634549350)

Another overall goal is better song representation, whether this be an arrow pose's mouth better fitting the heard vowel or the arrow directions / flow direction (in my opinion) better following the melody

Easy difficulties are typically restricted to either L, D, and R or L, U, and R but may sometimes step outside that comfort zone. This is so there is minimal vertical finger movement, and DEFINITELY nothing like having to move from D > U. As for U > D, such movement will be found very rarely in EZ. Flow will be mostly focused on D/U and R, as those are pressed by the index and middle fingers *(/!\ I AM LEFT HANDED /!\ I AM LEFT HANDED /!\)*, which should be the best at quicker movements.

Normal difficulties do use the whole four lanes, but unlike Hard they do not feature double notes. Nothing remarkable about NM; it should be about the same difficulty as base game's HD, and there's no deliberate difficulty added to the flow.

Hard difficulties can go all-out. And by all-out I mean some of the hardest things you'll see will be like the milf drop in 2hot hard except less bullshit to play on wasd. Flow breaks are often found (e.g. D > U and D + U) to spice up the chart.

As for the Erect difficulties, I honestly feel like there should never have been an underchart difficulty for this in the first place, and am inclined to just make ER identical to Nightmare... But Erect diff will probably just be no doubles and Nightmare will be all-out.

Anyway chat what do we think about this? What suggestions do any of you have to the format? This rechart *will* raise the overall difficulty of the game, but everything will still be fairly easy in the grand scope of rhythm games. My goal with these is, if they were the official charts, they would help FNF keeps its reputation as an introduction to rhythm games, but stop people from really thinking of it as the "baby's ddr" or whatever tf osu players say about fnf

Blammed Example: https://youtu.be/VMPikNypvHM

**Comment by dombomb64** [comment]
This actually sounds awesome. I would love to see this in the vanilla game.

**Comment by Poii0** [comment]
By the way, you could make this an issue, and select enhancement. Since the devs see it on there than here.

**Reply by isakube** [reply]
OH TRUE i forgot enhancement was still a label LOL. ill do it when i can fathom how to rewrite this as an enhancement, but i originally didnt wanna straightup ask FunkinTeam for a complete rechart of the game bc the game has already been through like two recharts and they're probably tired of hearing that their charts suck lol

**Reply by Poii0** [reply]
True LOL
